specification/confidentialledger/resource-manager/Microsoft.ConfidentialLedger/preview/2024-07-09-preview/examples/ConfidentialLedger_Create.json

```diff
@@ -24,7 +24,8 @@
         "hostLevel": "Info",
         "maxBodySizeInMb": 1,
         "subjectName": "CN=CCF Node",
-        "nodeCount": 3
+        "nodeCount": 3,
+        "writeLBAddressPrefix": "write"
```

Comment:
Please add a customization for the worker threads and limit it to 2. Default value is 0.

Reply:
added